src/azure-cli/azure/cli/command_modules/appconfig/_kv_import_helpers.py

```diff
@@ -1121,3 +1122,11 @@ def __flatten_key_value(key, value, flattened_data, depth, separator):
             )  # Ensure boolean values are properly stringified.
     else:
         flattened_data[key] = value if isinstance(value, str) else json.dumps(value)
+
+
+def __parse_json_file_data(json_string):
```

Comment:
Should we change this to be a json file instead of just a string. I don't think we have any use case here where a file is better.

Also, does the CLI not use type hinting?

Reply:
> Should we change this to be a json file instead of just a string.
Based on the usage, we can pass the file pointer here, but not the path.

> Also, does the CLI not use type hinting?

Currently, no